src/Directory.Build.props

```diff
@@ -48,16 +48,16 @@ Other Changes**
     <AkkaVersion>1.5.8</AkkaVersion>
     <RedisVersion>2.6.111</RedisVersion>
     <DockerVersion>3.125.15</DockerVersion>
-    <NetCoreTestVersion>netcoreapp3.1</NetCoreTestVersion>
+    <NetTestVersion>net7.0</NetTestVersion>
```

Comment:
Sense check here - Instead of maintaining testing against netcoreapp 3.1 I changed it to just using net7 and renamed the property. This is more inline with what is done on the main Akka project